pkg/build/api/types.go

```diff
@@ -42,8 +49,8 @@ type Build struct {
 // BuildSpec encapsulates all the inputs necessary to represent a build.
 type BuildSpec struct {
 	// ServiceAccount is the name of the ServiceAccount to use to run the pod
-	// created by this build.
-	// The pod will be allowed to use secrets referenced by the ServiceAccount
+	// created by this build. The pod will be allowed to use secrets referenced
```

Comment:
> what I was referring in my comment was what you suggested... "I would have cosmetic changes in separate commits"... by doing that I can't just click "Files changes" tab and do review work, but I will have to go each commit and filter out those I'm not interested in....

I'm not the only developer in the world who thinks there's a point to smaller commits. And it is not about "any" small commits, it's not about committing line by line, or other crazy stuff. There is logic behind it. One commit == one reason to change.
But I get that's not something that resonates well for other people in the team.

http://web.archive.org/web/20150325144256/http://www.conifersystems.com/2008/11/05/the-benefits-of-small-commits/
*One specific thing you should not do is combine cosmetic and functional changes in a single change.* (follow the link for further reasoning)

I'm talking about not putting functional and cosmetic changes in the same commit.
It's not about 1 x 30 commits, but 1 x 2 commits: one commit with the functional changes, one commit with all cosmetic changes you came across. You normally don't need to split the cosmetic changes just like you would split two unrelated functional changes.

> by doing that I can't just click "Files changes" tab and do review work, but I will have to go each commit and filter out those I'm not interested in....

That's not accurate. There would be no difference in the GitHub UI if you're looking at all file changes. Additionally, and optionally, you might want to look at each commit: one obviously only changing comments, etc; the other with "the meat".